drivers/SmartThings/matter-thermostat/fingerprints.yml

```diff
@@ -10,6 +10,13 @@ matterManufacturer:
     vendorId: 0x1396
     productId: 0x0043A
     deviceProfileName: air-purifier-hepa
+#Eve
+  - id: "4874/79"
+    deviceLabel: Eve Thermo
+    vendorId: 0x130A
+    productId: 0x004F
+    deviceProfileName: thermostat-heating-only-nostate
```

Comment:
Just a note that the `matter-thermostat` driver has dynamic profile naming, so when testing this we'll just want to confirm that the driver attempts to switch to this same profile when the device is onboarded. Should be easy to check by confirming that expected profile is still being used after the device is onboarded.